api/tests/e2e/orgs_test.go

```diff
@@ -219,25 +228,12 @@ var _ = Describe("Orgs", func() {
 				orgGUID = "nope"
 			})
 
-			It("returns a not found error", func() {
-				expectNotFoundError(resp, errResp, "Org")
-			})
-
 			AfterEach(func() {
-				orgGUID = originalGUID
+				deleteOrg(originalGUID)
```

Comment:
I guess we don't need this if the delete is deferred in the create?

Reply:
Are you thinking of the `createOrgWithCleanup` helper that we use in the repository tests? I don't think there's any deferred cleanup here.

Reply:
I was confused by the `createOrgWithCleanup`. I must pay more attention to the filenames!